testing/cloudbuild-templates/cloud-run-anthos-template.cloudbuild.yaml

```diff
@@ -0,0 +1,97 @@
+steps:
+- id: 'Generate UUID'
+  name: 'ubuntu'
+  entrypoint: '/bin/bash'
+  args:
+  - '-c'
+  - |
+    cat /dev/urandom | env LC_CTYPE=C tr -dc 'a-z0-9' | fold -w 15 | head -n 1 > _uuid
```

Comment:
This is one of the more cryptic bash commands I've seen in awhile, I think it needs a comment to explain what's happening and what we need it for.

Also, maybe we can simplify a bit to:

```
cat /dev/urandom | env LC_CTYPE=C tr -dc 'a-z0-9' | head -c 15
```

Not sure what `env LC_CTYPE=C` is for, never seen env used as part of an assignment so nice to learn! Shouldn't it be simpler `LC_CTYPE=C` without the `env`?

Reply:
comment added. Yes it works without `env`